src/pages/gateway/command-reference.md

```diff
@@ -11,7 +11,7 @@ All commands on this page support the `--help` argument, which provides informat
 
 ## aio api-manager:mesh:create
 
-Creates a new mesh based on the settings in the specified `JSON` file in your working directory. The `meshId` key value in your `JSON` file determines the name for your mesh. For more information, see [Creating a mesh].
+Creates a new mesh based on the settings in the specified `JSON` file in your working directory. After creating your mesh, you will receive a  `meshId`, like `continuing-aqua-bobby-2pUkDmZd`, to refer to it in the future. For more information, see [Creating a mesh].
```

Comment:
Everything looks good but can we remove these `meshIds` from the documentation? These could be real ones someday. Lets use some dummy ones.